spec/amber/controller/render.cr

```diff
@@ -0,0 +1,31 @@
+require "../../spec_helper"
+
+class TestController < Amber::Controller::Base
```

Comment:
Can you move this controller to the factories.cr file. It should be just a copy and paste.